starter_code/express-spotify-app/app.js

```diff
@@ -0,0 +1,76 @@
+const express = require('express');
+const path = require('path');
+const hbs = require('hbs');
+const app = express();
+//const fs = require('fs');
+//const bodyParser = require('body-parser');
+
+
+app.set('views', path.join(__dirname, 'views'));
+app.set('view engine', 'hbs');
+hbs.registerPartials(__dirname + "/views/partials");
+
+
+var SpotifyWebApi = require('spotify-web-api-node');
+
+// Remember to paste your credentials here
+var clientId = '6ec696df61634d928e76fc44cc12ffeb',
+    clientSecret = '74e2c61bdc0b40559fb319cc06763f61';
```

Comment:
**This are sensitive data**, so I recommend always save as a enviorment variable. We already learn this, so in your future project always add this kind of data into your `.env` file, and make sure that your `.env` file is mentioned in the `.gitignore` file before to make any commit :grin: 

Then I will told you a funny not so funny story about this :wink: